stdlib/Test/src/logging.jl

```diff
@@ -134,11 +134,13 @@ We can test the info message using
 If we also wanted to test the debug messages, these need to be enabled with the
 `min_level` keyword:
 
+    Using Logging
```

Comment:
```suggestion
    using Logging
```

Reply:
My bad. That was a silly error. Thanks for correcting it.